src/utils/triton.h

```diff
@@ -212,6 +258,90 @@ class TritonRequestView : public TritonRequestWrapper<TritonRequestView> {
   TRITONBACKEND_Request *handle_ = nullptr;
 };
 
+class TritonOutput {
+ public:
+  TritonOutput(TRITONBACKEND_Output *handle, uint64_t buffer_size) :
+      handle_(handle), buffer_size_(buffer_size) {}
+
+  OBufferDescr AllocateBuffer(device_type_t preferred_device, int preferred_device_id) const {
```

Comment:
Can this function be called many times on one output? Will it return the same buffer? If so, it should be called GetBuffer.